content/en/docs/instrumentation/js/instrumentation.md

```diff
@@ -935,6 +1006,50 @@ const counter = myMeter.createCounter('my.counter');
 cntr.add(1, { 'some.optional.attribute': 'some value' });
 ```
 
+### Configure Metric Views
+
+A Metric View provides developers with the ability to customize metrics exposed by the Metrics SDK. 
+
+To instantiate a view, one must first select a target instrument. The following are valid selectors for metrics: `instrumentType`, `instrumentName`, `meterName`, `meterVersion`, and `meterSchemaUrl`. Selecting by instrument name has support for wildcards, so you can select all instruments using `*` or select all instruments starting with `http` by using `http*`.
```

Comment:
I think it's a good idea to break out each of these selectors into bullet points.

I'd then call out that the name is a string that supports wildcards (rather than just implying that by saying there's wildcards).

Reply:
Updating